src/main/java/com/igot/cb/service/CbPlanServiceImpl.java

```diff
@@ -405,29 +436,44 @@ public ApiResponse updateCbPlan(ApiRequest request, String userOrgId, String tok
                                 throw new RuntimeException(e);
                             }
                         }
+                    }else {
+                        updatedCbPlanData.put(Constants.CONTEXT_DATA_REQUEST, cbPlanInfo.get(Constants.CONTEXT_DATA));
+                    }
+                    Map<String, String> userProfile = new HashMap<>();
+                    Map<String, Object> queryParams = Map.of(Constants.ID, userId);
+                    List<String> userFields = Arrays.asList(Constants.ROOT_ORG_ID);
+                    List<Map<String, Object>> userList = cassandraOperation.getRecordsByProperties(
+                            Constants.KEYSPACE_SUNBIRD, Constants.USER, queryParams, userFields, null);
```

Comment:
do not create unnecessary variables... 

List<Map<String, Object>> userList = cassandraOperation.getRecordsByProperties(
                            Constants.KEYSPACE_SUNBIRD, Constants.USER, 
                            Map.of(Constants.ID, userId), Arrays.asList(Constants.ROOT_ORG_ID), null);